### PR TITLE
Fix spelling mis-correction

### PR DESCRIPTION
**Pull request by keithc-ca**
* specimins -> specimens

### CODE REVIEW

File: runtime/gc_verbose_handler_realtime/VerboseHandlerOutputRealtime.cpp [hunk 1]
```diff
@@ -194,7 +194,7 @@ MM_VerboseHandlerOutputRealtime::handleInitializedInnerStanzas(J9HookInterface**
 void
 MM_VerboseHandlerOutputRealtime::handleEvent(MM_MetronomeIncrementStartEvent* eventData)
 {
-	/* if we are in a sync gc cycle, do nothing here and let the syncgc stanza handle things */
+	/* if we are in a sync gc cycle, do nothing here and let the sync gc stanza handle things */
```
Comment (pshipton) [inline runtime/gc_verbose_handler_realtime/VerboseHandlerOutputRealtime.cpp]: Dmitri specifically said this should remain as `syncgc`

Comment (pshipton) [reply runtime/gc_verbose_handler_realtime/VerboseHandlerOutputRealtime.cpp]: In https://github.com/eclipse/openj9/pull/5897#pullrequestreview-242318978

Comment (keithc-ca) [reply runtime/gc_verbose_handler_realtime/VerboseHandlerOutputRealtime.cpp]: I reverted this change.

**Comment by pshipton** [comment]
@jsoref fyi

**Pull request merged**